platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/isolated/IsolatedProjectsToolingApiCoupledProjectsIntegrationTest.groovy

```diff
@@ -333,21 +334,23 @@ class IsolatedProjectsToolingApiCoupledProjectsIntegrationTest extends AbstractI
             // some change
         """
         executer.withArguments(ENABLE_CLI, WARN_PROBLEMS_CLI_OPT, "-Dorg.gradle.internal.invalidate-coupled-projects=false")
-        def model3 = runBuildAction(new FetchCustomModelForEachProject())
+        def model3 = runBuildAction (new FetchCustomModelForEachProject())
 
         then:
         model3.size() == 2
         model3[0].message == "the message"
-        model3[1].message == "the message"
+        model3[1].message == "default message"
 
         and:
         fixture.assertStateUpdatedWithProblems {
             fileChanged("b/build.gradle")
             projectConfigured(":buildSrc")
-            projectsConfigured(":", ":a") // :a and :b are coupled, so configure a but reuse its model
+            projectsConfigured(":", ":b")
             modelsCreated(":b")
             modelsReused(":", ":a", ":c", ":buildSrc")
+            problem("Build file 'b/build.gradle': line 4: 'org.gradle.internal.invalidate-coupled-projects=false' is preventing configuration of project ':a'")
```

Comment:
I'm not sure I'd report this as a separate problem. It's really additional information for the real problem ("Cannot access project ':a' from project ':b'"), and seems to be saying basically the same thing, which is that there is direct access between the projects and so things might behave in a strange way if this is ignored.

If we do want to say something special in this case, I would include some additional information in the message of the real problem.